utils/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const newsApi = axios.create({
+  baseURL: "https://nc-news-l13l.onrender.com/api",
+});
+
+export function getArticles() {
+  return newsApi.get(`/articles`).then((res) => {
+    return res;
+  });
```

Comment:
If you narrow down what you return from the axios call then you won't need to destructure/access keys in the useEffect